### PR TITLE
 fix(ru): article <meta>

### PR DESCRIPTION
**Pull request by scffs**
1. translated comments
2. added title

### CODE REVIEW

File: files/ru/web/html/element/meta/index.md [hunk 1]
```diff
@@ -170,10 +170,10 @@ translation_of: Web/HTML/Element/meta
 ## Пример
 
 ```html
-<!-- In HTML5 -->
+<!-- В HTML5 -->
 <meta charset="utf-8">
 
-<!-- Redirect page after 3 seconds -->
+<!-- Переход на другую страницу через 3-и секунды -->
```
Comment (Saionaro) [inline files/ru/web/html/element/meta/index.md:176]:
```suggestion
<!-- Переход на другую страницу через три секунды -->
```

File: files/ru/web/html/element/meta/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: <meta>
+title: <meta> - тег для браузерной информации
```
Comment (Saionaro) [inline files/ru/web/html/element/meta/index.md:2]:
```suggestion
title: <meta> - элемент для описания метаданных
```

**Comment by Saionaro** [comment]
@scffs thank you for contributing to MDN!

**Comment by scffs** [comment]
thank you for review

**Pull request merged**